data/tutorials/gs_00_up_and_running.md

```diff
@@ -48,16 +48,20 @@ To install opam, you can [use your system package manager](https://opam.ocaml.or
 
 **For macOS**
 
+If you're installing with [Homebrew](https://brew.sh/): 
+
 ```shell
 # With Homebrew:
 $ brew install opam
+```
+
+Or if you're using [MacPorts](https://www.macports.org/):
 
+```shell
 # With MacPort:
 $ port install opam
 ```
 
-`# Homebrew` indicates the command if you're using [Homebrew](https://brew.sh/) as the installer, and `# MacPorts` is if you're using [MacPorts](https://www.macports.org/) to install.
-
 **For Linux**
 
 The preferred way to install opam is with your system's package manager on
```

Comment:
Suggestion:
```markdown
The easiest way to install opam is with your system's package manager on Linux (e.g., `apt-get install opam` or similar). [Details of all installation methods.](https://opam.ocaml.org/doc/Install.html). Depending on your package manager, you won't get the latest release of opam. If you want the latest release, consider installing it though the binary distribution. 
```

And in `**Binary Distribution*`:

```markdown
This method will fetch and initialize the latest version of opam, directly from the official servers:
```

Reply:
Done (I think) 🙂